app/src/main/java/co/touchlab/kampkit/android/BreedViewModel.kt

```diff
@@ -1,36 +1,41 @@
 package co.touchlab.kampkit.android
 
-import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
 import androidx.lifecycle.viewModelScope
 import co.touchlab.kampkit.db.Breed
 import co.touchlab.kampkit.models.BreedModel
+import co.touchlab.kampkit.models.DataState
 import co.touchlab.kampkit.models.ItemDataSummary
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
 import kotlinx.coroutines.flow.collect
 import kotlinx.coroutines.launch
 
 class BreedViewModel : ViewModel() {
 
     private var breedModel: BreedModel = BreedModel()
-    val breedLiveData = MutableLiveData<ItemDataSummary>()
-    val errorLiveData = MutableLiveData<String>()
+    private val _breedStateFlow: MutableStateFlow<DataState<ItemDataSummary>> = MutableStateFlow(
```

Comment:
I'm personally prefer `mutable` instead of `_` as a prefix, but we don't really have a specific rule here.